src/components/SessionInfoBar.tsx

```diff
@@ -10,6 +16,33 @@ export function SessionInfoBar() {
     store => store.session.serverInfo
   );
   const dispatch = useAppDispatch();
+  const [isHealthy, setIsHealthy] = useState(true);
+  const client = getClient();
+
+  const checkHeartbeat = async () => {
+    try {
+      let res: KintoResponse = await client.execute({
```

Comment:
Will this actually be a `KintoResponse`? Isn't this more a `Record<string, bool>`? (That probably isn't the right syntax, but you get what I mean)

Reply:
Yeah, technically execute will return `any`. But that endpoint in particular should return a hash table that is mostly bools (the details array being the exception). I'll make this more explicit.